src/consensus.rs

```diff
@@ -161,6 +162,7 @@ impl Consensus {
         .context("Failed to create timeout channel")?;
         let mut client = RaftClient::new(channel);
         let id = state_ref.this_peer_id();
+        // TODO: here ask who the first peer was and add it as the first voter
```

Comment:
why the TODO? isn't it what you've just done?

Reply:
Forgot to remove, thanks!